skimage/filters/_gaussian.py

```diff
@@ -155,3 +155,117 @@ def _guess_spatial_dimensions(image):
         return 3
     else:
         raise ValueError("Expected 2D, 3D, or 4D array, got %iD." % image.ndim)
+
+
+def difference_of_gaussians(image, sigma1, sigma2=None, *,
```

Comment:
I think here I would prefer `low_sigma` and `high_sigma`, or even `min_sigma` and `max_sigma`, even if they are conceptually a bit different from `blob_dog`.

Reply:
I am worried about "low" and "high" being confusing, e.g., are we talking about the high frequency cutoff (adjusted with the smaller sigma value) or the sigma value which is larger in magnitude?  "min" and "max" would cause less confusion, but it's weird to me to use the terms min and max for two values (although technically correct I suppose). What about `small_sigma` and `large_sigma`?

Reply:
I ended up going with `lesser_sigma` and `greater_sigma`. If you don't like that I can go with min/max.